src/webhooks/pubsub/slackNotifications.ts

```diff
@@ -113,12 +145,13 @@ export const constructSlackMessage = (
   productAreaToIssuesMap: Record<string, IssueSLOInfo[]>,
   now: moment.Moment
 ) => {
+  console.log(productAreaToIssuesMap);
```

Comment:
Maybe you're not ready for review yet. 😬

Reply:
Removed.